code/modules/jobs/access.dm

```diff
@@ -32,6 +32,12 @@
 	//if they are carying a card that has access, that works
 	else if(isliving(accessor))
 		var/mob/living/being = accessor
+		//humans can carry wallets that combine accesses so check for those as well.
+		if(ishuman(accessor))
+			var/mob/living/carbon/human/human_accessor = being
+			if(check_access(human_accessor.wear_id))
+				return TRUE
+		//check for id carried in hands, inside PDA's etc.
```

Comment:
```suggestion
	//if they are wearing a card that has access, that works
	else if(ishuman(accessor))
		var/mob/living/carbon/human/human_accessor = accessor
		if(check_access(human_accessor.wear_id))
			return TRUE
	//if they have a hacky abstract animal ID with the required access, let them in i guess...
	else if(isanimal(accessor))
		var/mob/living/simple_animal/animal = accessor
		if(check_access(animal.access_card))
```

Do we want to keep the old code from before?  Most of this code was for simple bots like medibots to have paramedic access.  Can you double check that it still works correctly?

Reply:
Oh wait, i just looked at the code and i realized the case is covered for when the player is carrying the ID in their active hand
![Screenshot (206)](https://github.com/tgstation/tgstation/assets/110812394/b4d0d849-7835-4ff4-8e53-ec06781baabf)
Yup your suggestion can be applied, never mind